nf_core/pipeline-template/.devcontainer/noop.txt

```diff
@@ -0,0 +1,3 @@
+This file copied into the container along with environment.yml* from the parent
```

Comment:
See other comment about removing the `environment.yml` file stuff.

Reply:
See my comment. Its for the enviroment for the codespaces. At this point not sued. its an file in the .devcontainers folder.